data/base.go

```diff
@@ -69,19 +69,22 @@ const (
 	//UserIDField is the userID
 	UserIDField = "userId"
 	//GroupIDField id the groupID
-	GroupIDField = "groupId"
-
-	deviceIDField = "deviceId"
-	uploadIDField = "uploadId"
-
+	GroupIDField         = "groupId"
```

Comment:
I would make two sets of constants if going this way, one for the URL parameters and one for the data model.  We don't want to mix-n-match in case we ever need to change one.  They are for two different things.